crates/http/src/config.rs

```diff
@@ -21,14 +21,14 @@ pub struct HttpTriggerConfig {
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
 #[serde(deny_unknown_fields, rename_all = "lowercase", tag = "type")]
 pub enum HttpExecutorType {
-    /// The component implements the Spin HTTP interface.
+    /// The component implements an HTTP based interface.
+    ///
+    /// This can be either `fermyon:spin/inbound-http` or `wasi:http/incoming-handler`
     #[default]
-    Spin,
+    #[serde(alias = "spin")]
+    Http,
```

Comment:
@lann  do you hate this?

Reply:
I considered hating it briefly but decided not to. In practice its really just "wagi or default" anyway.